README.md

```diff
@@ -1,3 +1,11 @@
+
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Run tests](https://img.shields.io/badge/Run%20tests-passing-green)
+![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=102)
```

Comment:
```suggestion
```

This resource is not being successfully fetched, so its best to take it out.

![image](https://user-images.githubusercontent.com/11148726/90989501-1c3d8080-e592-11ea-8927-03535fe381da.png)

Reply:
@vj-codes is this the correct link?

Reply:
ya sure @isabelcosta , so i shall remove both open source love and run tests badge, right ?

Reply:
yes please :) @CodeChamp-SS

Reply:
@isabelcosta made the required changes, please review :-)